appbuilder/services/livesync/livesync-service.ts

```diff
@@ -0,0 +1,102 @@
+///<reference path="../../../.d.ts"/>
+"use strict";
+
+import { exportedPromise } from "../../../decorators";
+
+export class ProtonLiveSyncService implements IProtonLiveSyncService {
+	private excludedProjectDirsAndFiles = ["app_resources", "plugins", ".*.tmp", ".ab"];
+
+	private get $liveSyncServiceBase(): ILiveSyncServiceBase {
```

Comment:
nope, the idea is to use the liveSyncServiceBase as dependency. It's not named very well, because it's not a base class. It has the basic functionality of livesync, but it cannot be used as base class for our livesync services.